rpms/pulp-scl/pulp-scl.spec

```diff
@@ -0,0 +1,153 @@
+# We specificall separate the 'pulp' and '-scl' parts from each other.
+# since spec file macros don't allow '-' and the auto generated ones
+# are based on %scl_name
+%global scl_name_base pulp
+%global scl_name_postfix -scl
+
```

Comment:
Why are we not using an scl_name_version to keep track of the version of our scl?